components/cache/cache_invalidation.rst

```diff
@@ -47,7 +47,7 @@ you can invalidate the cached items by calling
     // if you know the cache key, you can also delete the item directly
     $cache->delete('cache_key');
 
-Using tags invalidation is very useful when tracking cache keys becomes difficult.
+Using tag invalidation is very useful when tracking cache keys becomes difficult.
```

Comment:
Unsure here too

Reply:
It can either be stated as:

- Using tag invalidation..., or
- Using the invalidation of tags...